docs/guide/headless-cms.md

```diff
@@ -0,0 +1,177 @@
+# Using Apostrophe as a headless CMS
+
+Apostrophe is fully featured as a traditional CMS (with some non-traditional features). It is also immediately ready for use as a headless CMS. There are many scenarios where Apostrophe works well as a headless CMS, including:
```

Comment:
Apostrophe is fully featured as a traditional CMS. It is also immediately ready for use as a headless CMS. There are many scenarios where Apostrophe works well in a decoupled architecture, including: